app/app.go

```diff
@@ -1230,7 +1254,7 @@ func (app *App) GetIBCKeeper() *ibckeeper.Keeper {
 }
 
 // GetStakingKeeper implements the TestingApp interface.
-func (app *App) GetStakingKeeper() core.StakingKeeper {
+func (app *App) GetStakingKeeper() ibctestingtypes.StakingKeeper {
```

Comment:
don't see this method usage. can be removed?

Reply:
same about several other TestingApp-related methods nearby (GetScopedIBCKeeper, GetTxConfig)

Reply:
These are all methods for ibctesting stuff inside ibc-go to satisfy `ibctesting.TestingApp` interface

Reply:
Meaning we can't remove them